src/reducers/dataReducer.js

```diff
@@ -49,7 +48,7 @@ export function GRIDDLE_INITIALIZED(initialState) {
  * @param {Object} action - The action object to work with
 */
 export function GRIDDLE_LOADED_DATA(state, action) {
-  const transformedData = transformData(action.data);
+  const transformedData = transformData(action.data, state.get('renderProperties').toJSON());
```

Comment:
(I personally like the idea of throwing an error saying something like `given rowKey does not exist` so it's really clear why it's failing)